src/disco/css/Addon.scss

```diff
@@ -53,20 +53,7 @@ $addon-padding: 20px;
     }
   }
 
-  .Addon-install-button {
```

Comment:
The style is now controlled elsewhere for the most part so these were causing issues.